components/TravelPlanForm/index.js

```diff
@@ -127,38 +166,59 @@ const TravelPlanForm = ({ onFormSubmit }) => {
             type="date"
             value={endDate}
             onChange={(e) => setEndDate(e.target.value)}
+            required
           />
         </FieldContainer>
       </FlexContainer>
       <FlexContainer>
         <FieldContainer>
-          <Label>Activity:</Label>
+          <Label>Activity 1:</Label>
           <Input
             type="text"
-            value={activity}
-            onChange={(e) => setActivity(e.target.value)}
+            value={activities[0]}
+            onChange={(e) => updateActivity(e.target.value, 0)}
+            required
           />
         </FieldContainer>
+        <ActivityButton type="button" onClick={addActivity}>
+          Add Activity
+        </ActivityButton>
+      </FlexContainer>
+      {activities.slice(1).map((activity, index) => (
+        <FlexContainer key={index}>
```

Comment:
Array indices as keys is bad practice (as keys need to be unique throughout your whole page). Better construct something like   
```suggestion
        <FlexContainer key={`${activity}-${index}`}>
```